src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/OperationController.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer.Controllers
+{
+    [Route(ControllerName)]
+    [ApiController]
+    [HostRestriction]
+    [Authorize(Policy = AuthConstants.PolicyName)]
+    public class OperationController : ControllerBase
+    {
+        private readonly ILogger<OperationController> _logger;
+        private readonly IEgressOperationStore _operationStore;
+
+        public const string ControllerName = "operation";
+
+        public OperationController(ILogger<OperationController> logger, IEgressOperationStore operationStore)
+        {
+            _logger = logger;
+            _operationStore = operationStore;
+        }
+
+        [HttpGet("{operationId}")]
+        [ProducesWithProblemDetails(ContentTypes.ApplicationJson)]
+        [ProducesResponseType(typeof(Models.OperationStatusSuccess), StatusCodes.Status201Created)]
+        [ProducesResponseType(typeof(Models.OperationStatus), StatusCodes.Status200OK)]
+        public async Task<IActionResult> GetOperationStatus(Guid operationId)
+        {
+            return await this.InvokeService(async () =>
+            {
+                Models.OperationStatusBase status = await _operationStore.GetOperationStatus(operationId);
+                return new JsonResult(status)
+                {
+                    StatusCode = (int)(status.Status == Models.OperationState.Succeeded ? StatusCodes.Status201Created : StatusCodes.Status200OK)
+                };
+            }, _logger);
+        }
+
+        [HttpDelete("{operationId}")]
+        [ProducesWithProblemDetails(ContentTypes.ApplicationJson)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+        public async Task<IActionResult> CancelOperation(Guid operationId)
+        {
+            return await this.InvokeService(async () =>
+            {
+                await _operationStore.CancelOperation(operationId);
+                return Ok();
```

Comment:
Can you add a comment that CancelOperation will throw an exception if the operation ID doesn't exist? Reading it as-is, one might think it would return (and thus return 200) regardless of the existence of the operation ID.